opentelemetry-auto-instrumentation/src/opentelemetry/auto_instrumentation/__init__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
FYI it looks like this isn't included in the built docs.

Reply:
Sorry, which docs are the built docs?

Reply:
The ones you get running `tox -e docs` or `make html` in the docs dir.